src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -554,4 +702,26 @@ public void cannotOverrideRenewalWhenLoanDoesNotMatchAnyOfOverrideCases() throws
     return hasMessage(String.format("No item with barcode %s exists",
       item.getJson().getString("barcode")));
   }
+
+  private void createLoanPolicyAndSetAsFallback(LoanPolicyBuilder loanPolicyBuilder) throws
+    InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+    UUID policyId =
+      loanPoliciesFixture.create(loanPolicyBuilder).getId();
+    useLoanPolicyAsFallback(
+      policyId,
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.activeNotice().getId());
+  }
+
+  private void assertLoanHasActionComment(JsonObject loan, String actionComment) {
+    assertThat("loan should have 'actionComment' property",
+      loan.getString("actionComment"), is(actionComment));
```

Comment:
May want to make the string "actionComment" a constant.

Reply:
Done